packages/commonwealth/client/scripts/views/pages/notification_settings/index.tsx

```diff
@@ -498,6 +552,84 @@ const NotificationSettingsPage = () => {
             </div>
           );
         })}
+      <div style={{ marginTop: 26 }}>
```

Comment:
do not use inline style, add class and use css

Reply:
Unsure why this was there to begin with, not needed and removed in newest commit.